src/design-system/InputSelect.tsx

```diff
@@ -20,32 +22,46 @@ const SelectContainer = styled.div`
   margin-right: 8px;
 `;
 
-const SelectInput = styled.select`
-  -webkit-appearance: none;
-  -moz-appearance: none;
-  width: 100%;
-  border: none;
-  padding: 18px;
-  background: #e0e4e4;
-  outline: 0 solid transparent;
-  margin-top: 8px;
-  border-radius: 2px;
-  font-size: 16px;
-  color: #00413e;
+const caretSize = 5;
```

Comment:
Thanks for including this as a constant so that there are less magic numbers below.

- nit: Consider capitalizing this, since it is a constant. Lowercase is fine if you think this could become variable sometime in the future.
- ~Can you briefly explain what the value of `caretSize` represents and place it in a comment above?~ Nevermind, I just read the description and now I know what the caret is, I guess that's pretty obvious…

Reply:
I don't know whether we have really settled on a style for uppercasing constants (nothing linted at least, that I'm aware of) but if pressed I would favor [this rule](https://github.com/airbnb/javascript/#naming--uppercase), which advises against capitalizing here as well as in `MitigationInformation.tsx`